example/generated_secrets/redis-cluster.yaml

```diff
@@ -0,0 +1,44 @@
+---
+apiVersion: redis.redis.opstreelabs.in/v1beta1
+kind: RedisCluster
+metadata:
+  name: shubham
```

Comment:
Please correct the name of cluster and also I cannot see the new block in the example